actions/tag-and-release/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Tag and release"
+description: "Tag and release your repo with target tag."
+
+inputs:
+  tag:
+    description: "The version of the tag to be publish and released e.g 1.0.0"
+    required: true
+  release-title:
+    description: "Text for the github release title"
+    required: true
+  release-body:
+    description: "Text for the github release body (this describes the release content / Changelog)"
+    default: ""
+    required: false
+  github-username:
+    description: "The GitHub username for committing the changes."
+    required: true
+  github-email:
+    description: "The GitHub email for committing the changes."
+    required: true
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Create tag
+      run: |
+        git config --local user.name ${{ inputs.github-username }}
+        git config --local user.email ${{ inputs.github-username }}
+        git tag v${{ inputs.tag }}
```

Comment:
I am wondering whether or not we should hardcode the `v` prefix.. 🤔

Reply:
I really don't like the v on the tags

Reply:
me neither 😬
what's your opinion @yannick-roeder @philipp94831?

Reply:
I also don't like it

Reply:
example now https://github.com/bakdata/github-playground/releases/tag/2.0.7